npc/custom/warper.txt

```diff
@@ -526,6 +526,707 @@ S9: Restrict("RE");
 	Go("paramk",97,17);
 S10: Go("turbo_room",99,114);
 
+OnNaviGenerate:
```

Comment:
This could do with an explanation in https://github.com/rathena/rathena/blob/master/doc/script_commands.txt#L849